admin/app/views/workarea/admin/shipping_skus/edit.html.haml

```diff
@@ -0,0 +1,59 @@
+- @page_title = t('workarea.admin.shipping_skus.edit.title', name: @sku.name)
+
+.view
+  .view__header
+    .grid
+      .grid__cell.grid__cell--25
+        / left side element needed for spacing
```

Comment:
I know this looks a bit odd, but I really couldn't think of a better approach without diving too much into the CSS of `.grid`. Maybe something we can revisit later, but adding this spacer element is what prevents this:

<img width="995" alt="Screen Shot 2020-08-27 at 9 18 07 AM" src="https://user-images.githubusercontent.com/113026/91447325-8b42ff80-e846-11ea-91da-8099fc851b20.png">

from looking like this:

<img width="999" alt="Screen Shot 2020-08-27 at 9 17 57 AM" src="https://user-images.githubusercontent.com/113026/91447352-91d17700-e846-11ea-9677-037ad39d5497.png">

That looks like a bug on the inventory SKUs UI, but maybe I can address that in a later ticket...

Reply:
I believe you can fix this by adding the `grid--right` modifier to the grid instead of an extra `grid__cell`.